index.html

```diff
@@ -9035,6 +9041,8 @@ <h2>Screen capture</h2>
  and a <a>command</a> <a>Take Element Screenshot</a>
  for doing the same with the visible region of an <a>element</a>’s
  <a>bounding rectangle</a> after it has been <a>scrolled into view</a>.
+ It also provides a <a>Take Full Page Screenshot</a> <a>command</a>
```

Comment:
Please move this part up so it's listed before `Take Element Screenshot`, and continues the sentence.

Reply:
Done.